code/modules/tools/tool_item.dm

```diff
@@ -19,7 +19,7 @@
 
 	if(. == TOOL_USE_SUCCESS)
 		var/decl/tool_archetype/tool_archetype = GET_DECL(archetype)
-		to_chat(user, SPAN_NOTICE("You finish [success_message || tool_archetype.use_message] \the [target] with \the [src]."))
+		user.visible_message(SPAN_NOTICE("[user] finishes [success_message || tool_archetype.use_message] \the [target] with \the [src]."))
```

Comment:
Won't the tenses of `use_message` need changing now?

Reply:
Shouldn't, in most cases.

Reply:
Yeah I was dumb, hid the question.